languages/de-DE/Help/import.tid

```diff
@@ -0,0 +1,32 @@
+title: $:/language/Help/import
+description: Importiert mehrere Tiddler aus einer Datei
+
+Diese Befehl importiert / extrahiert Tiddler aus folgenden Dateien: 
+
+* TiddlyWiki `*.html`
+* `*.tiddler`
+* `*.tid`
+* `*.json`
+* oder andere local `text` Dateien
+
+Der `<deserializer>` Parameter muss angegeben werden. Anders als beim `--load` Befehl, der diese Information aus der Datei Endung erraten kann. 
```

Comment:
muss angegeben werden - anders als ... **von** der **Datei-Endung / Dateiendung** **ableitet**.

Reply:
ableiten. ... ja. 

Das problem ist allerdigs, dass es sich in Wirklichkeit um "raten" handelt. ... Ich werds aber ändern.